salt/addons/init.sls

```diff
@@ -35,10 +35,11 @@ deploy_addons.sh:
   http.wait_for_successful_query:
```

Comment:
That's not a bad idea... these were all originally in kubernetes-master, and the check was moved as part of the split. Might be best to put it with kube-apiserver.

However, waiting on the service wouldn't be enough... You can wait on the whole state though with `require: [kube-apiserver]' - which will wait on the service, cmd.run, etc.